config/database.yml

```diff
@@ -20,6 +20,10 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: localhost
+  username: postgres
+  password: root
+  timeout: 5000
 
```

Comment:
@henask12 This is something that should only be in your local environment buddy, I would appreciate if you do add it to the `.gitignore` file before merging this branch, other wise every time I pull or you pull the original repo our db settings will break!